File: 2.0/java/keywords/findKeywords.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.keywords.model.Keyword;

import java.util.Arrays;
import java.util.List;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        List<Keyword> keywords = client.keywordsApi().find(Arrays.asList("KEYWORD1", "KEYWORD2"));
    }
}

File: 2.0/csharp/keywords/findKeywords.cs
using System.Collections.Generic;
using CallfireApiClient;
using CallfireApiClient.Api.Keywords.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        List<string> keywords = new List<string> {"SUN", "MOON"};
        IList<Keyword> avaialbeKeywords = client.KeywordsApi.Find(keywords);
    }
}
